packages/next/src/build/index.ts

```diff
@@ -2116,31 +2121,29 @@ export default async function build(
                             isSSG = true
                             isStatic = true
 
-                            appStaticPaths.set(originalAppPath, [])
-                            appStaticPathsEncoded.set(originalAppPath, [])
+                            paths.static.set(originalAppPath, [])
                           }
 
                           if (
-                            workerResult.encodedPrerenderRoutes &&
-                            workerResult.prerenderRoutes
+                            workerResult.generatedRoutes &&
```

Comment:
What's motivating this change of calling them "generated routes" instead of "prerendered routes"? Does "generated routes" include some other kind of route considering it seems like we do still use the concept of a "prerendererd route" via `prerenderManifest.routes`?

Reply:
We can rename them to prerendered routes. During refactoring it was easier to find places where problems were happening with a rename instead.